jenkins_jobs/trigger_host_os_iso_build_from_versions_repo/script.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+set -x
+
+
+TARGET_CONTEXT="Build Host OS"
+
+
+get_build_state(){
+    # this will evaluate the queried keys as variables: state=<state>,
+    # target_url=<url>
+    eval $(github_api "$GITHUB_USER_NAME" "$GITHUB_PASSWORD" \
+                      query_status "${GITHUB_ORGANIZATION_NAME}/versions" \
+                      "$ghprbPullId" "$TARGET_CONTEXT" --state \
+                      --target-url || echo "exit 1")
```

Comment:
I'd avoid using positional arguments. Also, IMHO using a fields/attributes/keys argument with desired keys in comma-separated format would be simpler. Eg:

github_api --user $GITHUB_USER_NAME --password $ GITHUB_PASSWORD
      query_status --repo "${GITHUB_ORGANIZATION_NAME}/versions" 
      --pull-request "$ghprbPullId" --context "$TARGET_CONTEXT" --fields "state,target-url"